core/transformer/src/IModelImporter.ts

```diff
@@ -487,6 +487,17 @@ export class IModelImporter implements Required<IModelImportOptions> {
       }
     }
   }
+
+  /** Examine the geometry streams of every [GeometricElement3d]($backend) in the target iModel to identify [GeometryPart]($backend)s that each have exactly
+   * one reference to them. Update the geometry stream of each referencing element to embed the part's geometry instead, then delete the part.
+   * @note This method is automatically called from [[IModelTransformer.processChanges]] and [[IModelTransformer.processAll]] if
+   * [[IModelTransformOptions.inlineGeometryPartReferences]] is `true`.
+   */
+  public inlineGeometryPartReferences(): void {
```

Comment:
nit: this name sounds like it implies all part references get inlined. Is the intention more like `heuristicallyTryInlineParts` or just `optimizeGeometry` if the future additional inlining-based geometry optimizations mentioned in the description would also be performed through this call.

Reply:
I renamed it to "optimizeGeometryPartReferences"; in future it might take optional arguments to configure different optimizations.

Reply:
maybe this PR should deprecate `simplifyElementGeometry` and instead you should add `replaceBRepsWithMeshes` to `OptimizeGeometryOptions`?

Reply:
I don't think so. simplifyElementGeometry is applied only to the elements that are actually transformed, on a per-element basis. The new option is a post-processing step applied to all elements in the target iModel.

Reply:
ok yeah let's not mix them then.